pkg/sensors/program/map.go

```diff
@@ -104,6 +104,10 @@ type Map struct {
 	Owner        bool
 }
 
+// GlobalMaps keeps a record of all global maps to exclude them from per policy
+// memory map accounting.
+var GlobalMaps map[string]bool
```

Comment:
should I put this behind a lock or not?

Reply:
yes, please. And can we hide the global object and have functions to access it from other packages?


PS: I wonder if there is a better solution that keeping a global map.
PPS: Don't we need to delete maps from the map? I guess currently all global maps are part of the base sensor so it shouldn't matter, but maybe in the future we want global maps that are added/deleted at runtime (?)

Reply:
oh yeah it's better behind locks, and indeed I needed to add delete, I moved the logic to add in Load instead of at creation and delete at Unload! :) Please take a look again at that commit!

Reply:
> PS: I wonder if there is a better solution that keeping a global map.

On that I don't know :/ we don't have a central object that keeps references on maps so I guess that might be the only way?